resources/tomcat-config/crafter/studio/extension/studio-config-override.yaml

```diff
@@ -195,3 +199,25 @@ studio.mail.port: @SMTP_PORT@
 # studio.mail.smtp.ehlo: true
 # Enable/disable (value true/false) debug mode for email service. Enabling debug mode allows tracking/debugging communication between email service and SMTP server.
 # studio.mail.debug: false
+##################################################
+##        Cluster node registration             ##
+##################################################
+# Cluster node registration data
+# studio.clustering.node.registration:
+    # cluster member local IP
+#   localIp: 127.0.0.1
```

Comment:
Change the example to become `10.1.1.150` to avoid people leaving it on loopback